enhancements/baremetal/enable-baremetal-on-aws.md

```diff
@@ -0,0 +1,148 @@
+
+## Summary
+
+Baremetal Host API is available when deploying an OpenShift cluster with the baremetal platform (via the IPI or AI (Assisted Installer) workflow) or with the on-premise platforms Openstack, vSphere, and None.
+Adding the possibility to deploy and manage bare metal hosts from clusters not in an on-premise infrastructure would be beneficial to customers.
+
+## Motivation
+
+Having the control plane on a pure cloud platform, allowing the deployment and management of bare metal workers, provides a true hybrid cloud solution.
+
+### Goals
+
+Support the centralized host management use case by enabling Baremetal Host API on AWS platform, in order to allow clusters deployment when running on the AWS platform.
+
+### Non-Goals
+
+Allow Baremetal Host API to be fully enabled on other cloud platforms, such as GCP and Azure.
+
+## Proposal
+
+The BMO (baremetal-operator) provides the Baremetal Host API, which is configured and managed by the CBO (Cluster Baremetal Operator).
+
+Currently CBO checks the platform and if it is not baremetal, Openstack, vSphere, or None, it will be in a "disabled" state i.e. it will
+1. set status.conditions Disabled=true and
+2. not read or process the Provisioning CR and thus not deploy baremetal-operator.
+
+This proposal is to allow CBO to be enabled on AWS platform.
+
+To keep the testing matrix as it is, the allowed configuration options
+of the Provisioning CR will be kept restricted to exactly those required by centralized host management.
+
+*Only spec.provisioningNetwork=Disabled mode will still be accepted in the Provisioning CR.*
+
+If any other provisioningNetwork mode is set, the CBO webhook will refuse the change in the usual way.
+
+Notes:
+
+1. when the Provisioning CR is set to provisioningNetwork=Disabled mode, worker nodes would be booted via virtual media. This removes the requirement for the Provisioning Network which can be expected to be available only in Baremetal platform types.
+
+2. documentation will need to be added to the centralized host management documentation explaining how to create and update a Provisioning CR for the AWS platform.
+
+### Workflow Description
+
+#### Variation
+
+### User Stories
+
+#### Story 1 - Current IPI baremetal platform use case
+
+No change.
+
+#### Story 1 - centralized host management use case
+
+As a user of a hub cluster that performs central infrastructure management, and optionally zero-touch provisioning, I need to provision hosts using the k8s-native API (Baremetal Hosts CR) even when the hub cluster has a platform of AWS.
+
+### Risks and Mitigations
+
+One concern of this implementation is that the ip address of the BMCs of the bare metal hosts deployed and managed in this way will be exposed externally, as they need to communicate with the control plane, hosted in the AWS platform.
+To reduce the risk is highly advised to use some vpn/tunnel-like connection between the bare metal hosts and the control plane.
```

Comment:
I'd go so far as to say this is essential, and the risk is that we don't have a way to verify that it has been done correctly.